dojo/filters.py

```diff
@@ -235,6 +235,35 @@ def filter(self, qs, value):
         return self.options[value][1](self, qs, self.field_name)
 
 
+class FindingHasJIRAFilter(ChoiceFilter):
+    def no_jira(self, qs, name):
+        return qs.filter(Q(jira_issue=None) & Q(finding_group__jira_issue=None))
+
+    def any_jira(self, qs, name):
+        return qs.filter(~Q(jira_issue=None) | ~Q(finding_group__jira_issue=None))
+
+    def all_items(self, qs, name):
+        return qs.filter(Q(jira_issue=None) | Q(finding_group__jira_issue=None))
```

Comment:
This will only return Findings that are either missing `jira_issue` or missing `finding_group__jira_issue`. Shouldn't it simply return everything, unfiltered?

Reply:
This function is only called when the input supplied does not match what is expected, and low key sorta fails. I believe the thought process is along the lines of "if something unexpected happens, removing all findings with jira links would be a good indicator. This is just me speculating though. Would need @hblankenship to confirm

On the other side, returning all results (as if the filter was never applied) is a totally valid approach as well

Reply:
I apologize for not getting to this earlier. Testing shows that the option returns the same thing not applying the filter returns. The reason it returns them all, despite what it looks like, is because jira_issue is None for finding_group__jira_issue findings and finding_group__jira_issue is None for jira_issue items.  I have modified it to just return without filtering to alleviate the confusion.